tests/invite/deleteInvite.ts

```diff
@@ -14,9 +14,9 @@ Deno.test({
     // Assertions
     assertExists(invite);
 
-    const deletedInvite = await bot.helpers.deleteInvite(invite.code);
+    await bot.helpers.deleteInvite(invite.code);
 
     // Assertions
-    assertExists(deletedInvite);
```

Comment:
Very bad idea, this is throwing a request at the api every 100ms until it magically appears

Reply:
deleteInvite() returns `Promise<void>` so I use assertRejects getInvite() (getInvite throws an error if not found)
what do you mean by "this is throwing a request at the api every 100ms until it magically appears"

Reply:
I misread, sorry was tired. Looks good